packages/playground/wordpress-builds/build/Dockerfile

```
@@ -57,6 +57,11 @@ RUN cp -r wordpress wordpress-static && \
 # Remove the sqlite database integration from the static files
 RUN rm -rf wordpress-static/wp-content/mu-plugins
 
+# List assets that will be available to remotely so Playground can judge whether
+# to request a remote asset or delegate the request for a missing file to PHP.
+# TODO: What is a better name for this file?
+RUN find wordpress-static -type f | sed 's#^wordpress-static/##'> playground-remote-asset-paths
```

Maybe `wordpress-remote-asset-paths`?

```suggestion
RUN find wordpress-static -type f | sed 's#^wordpress-static/##'> wordpress-remote-asset-paths
```

---

But `playground-remote-asset-paths` also works.

---

🤔 The main point was to communicate that this is a special setup for minified WP builds, and it seemed a playground-specific thing to me. If I think about someone listing the files in one of the zips, I think it is clearer to see "playground-" than "wordpress-" when they ask "What is this thing?".

I was on the fence about this, but after talking it out, I think I'll leave it alone unless someone else feels strongly otherwise.

---

Heh, I save this comment and then think the opposite. Within playground, I would rather read `/internal/wordpress-remote-asset-paths`, so let's go with that.

---

Note this is only useful on the web and only for some WordPress builds but not for others. Also, which WordPress build is running may change in runtime.

---

OK, so it sounds like it belongs with some kind of Wordpress-associated metadata.

I'll start by acting as if WP cannot be swapped out and address the request handling behavior and then address the possibility of WP switching, all within the scope of this PR.

---

> the "active" WordPress build may change in runtime.

@adamziel it looks like the web interface reloads the page every time the config changes. Where can the "active" WordPress build change anytime during runtime? Is this something that can happen with wp-now without reloading everything? (Planning to look further but wanted to get the question out there)